src/atomsbase.jl

```diff
@@ -11,17 +10,20 @@ Construct an AtomsBase `FlexibleSystem` from a Chemfiles Frame.
 function AtomsBase.FlexibleSystem(frame::Chemfiles.Frame)
     atoms = map(1:length(frame), frame) do i, atom
         pos = Chemfiles.positions(frame)[:, i]u"Å"
+        velocity_arg = ()
         if Chemfiles.has_velocities(frame)
-            velocity = Chemfiles.velocities(frame)[:, i]u"Å/ps"
-        else
-            velocity = zeros(3)u"Å/ps"
+            velocity_arg = (Chemfiles.velocities(frame)[:, i]u"Å/ps", )
+        end
+
+        species = AtomsBase.ChemicalSpecies(Chemfiles.atomic_number(atom))
```

Comment:
in case where the atomic number is unknown, this function will return 0. Does `AtomsBase.ChemicalSpecies` handle this?

Reply:
Do you think it is more reliable to construct a species based on `Chemfiles.atomic_number` or based on `Chemfiles.type`, i.e. what is more likely to be available ?

Reply:
After some experimentation I think this is more reasonable. Because otherwise "custom" element symbols get passed to atomsbase, which would not understand it anyways.